content/blog/how-to-generate-microsoft-dsc-adapted-resource-manifests-with-dscresource.authoring.md

```
@@ -0,0 +1,440 @@
+```
+
+## Wrap-up
+
+You've just learned that adapted resource manifests make class-based DSC resources easier for Microsoft DSC to discover, understand, and execute. They remove the guessing game, especially around the two PowerShell adapters. When the time was measured, it made visible that adapted resource manifests were way quicker by short-circuiting the process in the engine.
+
+But the important part is that you should not have to maintain those files by hand. Especially because there's already a ton of automation around DSC community modules. The `DscResource.Authoring` module was brought to life using this information in existing repositories. From there, it can be hooked into the existing build workflows with ease.
+
+If you maintain a DSC community module with class-based resources, or you've now been inspired to create them, then this next step can be added simply. The short version:
+
+1. Open the `build.yaml`
+2. Choice one of the two tasks exposed from `DscResource.Authoring`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)